crypto/AddressCalc.c

```diff
@@ -25,10 +30,6 @@ int AddressCalc_addressForPublicKey(uint8_t addressOut[16], const uint8_t key[32
     if (addressOut) {
         Bits_memcpyConst(addressOut, hash, 16);
     }
-    return hash[0] == 0xFC;
+    return AddressCalc_validAddress(addressOut);
```

Comment:
This seems to be the only actual change -- could you remove the rest from the patch?